libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/ClientManager.java

```diff
@@ -381,8 +381,6 @@ public Bundle createNewAccount(String accountName, String username, String refre
             // WARNING! This assumes all user data is a String!
             accountManager.setUserData(acc, key, extras.getString(key));
         }
-
-        accountManager.setAuthToken(acc, AccountManager.KEY_AUTHTOKEN, authToken);
```

Comment:
This is a useless call, because ```AccountManager``` automatically caches auth token in memory anyway.